apps/etterlatte-behandling/src/main/kotlin/behandling/tilgang/TilgangRoutes.kt

```diff
@@ -17,7 +17,7 @@ internal fun Route.tilgangRoutes(sakService: SakService, sakServiceAdressebeskyt
     route("") {
         post("/person") {
             val fnr = call.receive<String>()
-            val harTilgang = !sakService.sjekkOmSakHarStrengtFortroligBeskyttelse(fnr)
```

Comment:
Nå er det kanskje greit å kalle det adressebeskyttet i stedet for gradert?

Reply:
Ja - i grunn enig